cost/google/rightsize_vm_recommendations/google_rightsize_vm_recommendations.pt

```diff
@@ -0,0 +1,666 @@
+name "Google Rightsize VM Recommender"
+rs_pt_ver 20180301
+type "policy"
+short_description "This Policy finds Google Machine Type (Rightsize) Recommendations and reports when it finds them. See the [README](https://github.com/flexera-public/policy_templates/tree/master/cost/google/idle_vm_recommendations) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
```

Comment:
README URL still links to idle_vm_recommendations

Reply:
Done